backend/src/main/java/de/neuefische/backend/service/RecipeService.java

```diff
@@ -48,4 +48,17 @@ public RecipeNormalized saveNewRecipe(RecipeDto recipeDto) {
         return new RecipeNormalized(temp);
     }
 
+    // This method is not used in the frontend. It's only used as help function, because of the Normalized enums.
```

Comment:
```suggestion
```

you can delete the comment, because it is clear what the method is used for and needs no description 💯